Program.cs

```diff
@@ -1,24 +1,94 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Logging;
-
-namespace PhoneEdit
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using MySqlConnector;
+using PhoneEdit.Data;
+
+var builder = WebApplication.CreateBuilder(args);
+
+try
+{
+    var identityContextSecrets = builder.Configuration["ConnectionStrings:IdentityContext"] ?? 
+                                 throw new InvalidOperationException("Connection string 'IdentityContext' not found.");
+    var phoneBookContextSecrets = builder.Configuration["ConnectionStrings:PhoneBookContext"] ?? 
```

Comment:
builder.Configuration.GetConnectionString() is preferable in both cases 

also, you could provide sample strings in Appsettings.Development.json